lib/eventlog/src/evtlog.h

```diff
@@ -84,9 +84,11 @@
 #define EVT_FAC_LOCAL7      (23<<3) /* reserved for local use */
 
 #ifdef __GNUC__
-#define EVT_GNUC_PRINTF_FUNC(format_idx, first_arg_idx)  __attribute__((format(printf, format_idx, first_arg_idx)))
+#define EVT_FORMAT_PRINTF_FUNC(format_idx, first_arg_idx)  __attribute__((format(printf, format_idx, first_arg_idx)))
```

Comment:
We are using `glib`, that actually defined this macro: `G_GNUC_PRINTF`. I think it would easier to use that (and possibly more compatible).

Reply:
The above macro `G_GNUC_PRINTF` is going to work with clang also, as clang pretend to be gcc `4.x` so defines `__GNUC__`.

Reply:
Oh and if it is not clear, so get ride of those macros and use this one directly.

Reply:
Thank you Kokan, this clears up a lot of copy pasted code. I will change it as you asked.